src/js/components/AppListComponent.jsx

```diff
@@ -455,6 +456,27 @@ var AppListComponent = React.createClass({
 
     var totalColumnSpan = 8;
 
+    var statusDocsURL = "https://mesosphere.github.io/marathon/docs/" +
+      "marathon-ui.html#application-status-reference";
+    var statusTooltipMessage = (
+      <span>
+        At-a-glance overview of the global application or group state.
+        Includes Running, Deploying, Suspended, Delayed and Waiting.
+        <a href={statusDocsURL} target="_blank">Read more</a>.
+      </span>
+    );
+
+    var healthDocsURL = "https://mesosphere.github.io/marathon/docs/" +
+      "marathon-ui.html#application-health-reference";
+    var healthTooltipMessage = (
+      <span>
+        General health condition of the application tasks.
+        Includes Healthy, Unhealthy, Unknown, Staged, Overcapacity and
+        Unscheduled.
+        <a href={healthDocsURL} target="_blank">Read more</a>.
+      </span>
```

Comment:
Not a big fan of having these strings as variables. Can we either move them to a constants file or include them directly in the `render` method please?